### ISSUE TITLE
Unable to rename files

### ISSUE THREAD

**Issue by daninsky1** [linuxmint/nemo#3155]
```
 * Nemo version 5.2.4
 * Windowed nemo
 * Distribution - Ubuntu MATE
 * Nvidia driver
 * 64 bit
 ```

**Issue**
Impossible to rename file

**Steps to reproduce**
Press F2, or right click menu and rename, try to modify the name and the first letter will close the input immediately and confirm the edition with the first letter

**Expected behaviour**
Keep the input open until you press enter or out click to cancel

**Other information**

**Comment by Jeremy7701** [comment]
Is this true for **all** files?
Including freshly created files?
You won't be able to rename files unless you have valid rights to do so.

**Comment by daninsky1** [comment]
This is true for all files, even freshly created. Files are renamed, but only the first letter that is typed registers in the file/dir name.

**Comment by JulianGro** [comment]
This issue has just started happening for me on nemo 5.6.4 on Debian Testing.
It has never happened before.
I am fairly sure that I didn't update anything related to nemo.

**Comment by awvalenti** [comment]
Probably the same issue I found today. I investigated it and fount out that it's related to the scrollbar. It only happens when scrollbar is near being displayed. In other words, if it would be displayed in case a new file or folder was created.

GIF below demonstrates it:
![Image](https://github.com/user-attachments/assets/02293af6-9904-41bf-8019-61c375f43f29)

**Comment by awvalenti** [comment]
Above, I'm using keys:
- Ctrl Shift N to create new folders
- F2 to rename them
- Enter to commit folder name (when the bug does not happen)

**Comment by awvalenti** [comment]
Steps to reproduce:
1. Open a nemo window with an empty folder
2. Create 3 folders inside it
3. Size the window as to fit 2x2 items, but so the second row gets slightly hidden and a very initial scrollbar appears
4. Try to rename anything
5. You won't be able to. Any key you press will commit the name.

![Image](https://github.com/user-attachments/assets/874f261f-b118-4ba3-a77a-e6a217950ff2)

**Comment by NickNackGus** [comment]
I'm able to reproduce this in Nemo 6.6.3 on Mate 1.28.0. At least now I know it's an issue with the scrollbar just barely appearing - which explains why it's happening consistently for a folder in one of my monitors (full screen), but not the other; it's a different size. This bug has gone back for years (https://github.com/linuxmint/nemo/issues/2030).
